src/etl/ETLService.h

```diff
@@ -152,6 +152,17 @@ class ETLService
         return ledgerPublisher_.lastCloseAgeSeconds();
     }
 
+    /**
+     * @brief Check for the amendment blocked state.
+     *
+     * @return true if currently amendment blocked; false otherwise
+     */
+    bool
```

Comment:
with this , our continuously logging " amendment not support " should  be easy~ yeah!